### DISCUSSION TITLE
Volume control?

### DISCUSSION THREAD

**Discussion by W3AXL** [chuot/rdio-scanner#152] [Q&A]
Curious if there's any kind of volume control available to the user? I've noticed the scanner is pretty much the loudest tab I've got open and it would be nice to turn it down a bit, or have volume slider somewhere in the UI to do that on-the-fly.

**Comment by UberPlexCa** [comment]
https://chrome.google.com/webstore/detail/volume-master/jghecgabfgfdldnmbfkhmffcabddioke

**Reply by W3AXL** [reply]
I suppose that is one solution. Howeve, it would be nice to have a native volume control so you don't need to install a third-party extension.

I may look into the source code and see how difficult it would be to add in a small slider to the UI. After all, a real radio scanner would absolutely have volume control :)

**Reply by mba707** [reply]
Agree, using a browser extension is far from ideal. Please post here if you find a way to add. And audio compressor and EQ would be especially nice.

**Reply by Mark02420** [reply]
I would also like to see a volume control.  I would prefer that a volume  be put between the time and queue number in the display. If that’s not possible it could be put right above the time. My use case is that I have multiple virtual scanners running in my browser and each of them are minimized to show only the top half of the display window with the information about the current call being played. This usually works great except when things get real busy on a particular channel.  I used to always have two or three actual scanners running at a time and I was able to turn up or down the individual volumes of the scanners when needed.

**Comment by rashnet** [comment]
I was wondering the same thing, it would be nice to have a volume slider on the webpage to control the volume.

**Comment by derpotts** [comment]
Any update on the volume issue?
